OpenRA.Mods.Common/EditorBrushes/EditorDefaultBrush.cs

```diff
@@ -39,6 +39,7 @@ public class EditorDefaultBrush : IEditorBrush
 		readonly EditorViewportControllerWidget editorWidget;
 		readonly EditorActorLayer editorLayer;
 		readonly Dictionary<int, ResourceType> resources;
+		int2 worldLocation;
```

Comment:
`worldLocation = worldRenderer.Viewport.ViewToWorldPx(mi.Location)` makes it world view or projection coordinates.
I could rename it `worldViewLocation`.